rpc/backend/blocks.go

```diff
@@ -91,6 +91,36 @@ func (b *Backend) GetBlockByNumber(blockNum rpctypes.BlockNumber, fullTx bool) (
 	return res, nil
 }
 
+// GetBlockReceipts returns a list of Ethereum transaction receipts given a block number
+func (b *Backend) GetBlockReceipts(blockNum rpctypes.BlockNumber) ([]map[string]interface{}, error) {
+	resBlock, err := b.TendermintBlockByNumber(blockNum)
+	if err != nil {
+		return nil, nil
```

Comment:
why not returns the error details?

Reply:
after checking, I am reverting to return nil there

The reason is that geth is not returning any error when a block is not found so we need to return nil if we want to match the behavior